<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockCornerSettingsDialog</class>
 <widget class="QDialog" name="DockCornerSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dock Corner Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Top Left Corner</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="topLeftTopButton">
          <property name="text">
           <string>Top Dock Area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="topLeftLeftButton">
          <property name="text">
           <string>Left Dock Area</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Top Right Corner</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="topRightTopButton">
          <property name="text">
           <string>Top Dock Area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="topRightRightButton">
          <property name="text">
           <string>Right Dock Area</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Bottom Left Corner</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="bottomLeftBottomButton">
          <property name="text">
           <string>Bottom Dock Area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bottomLeftLeftButton">
          <property name="text">
           <string>Left Dock Area</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Bottom Right Corner</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="bottomRightBottomButton">
          <property name="text">
           <string>Bottom Dock Area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bottomRightRightButton">
          <property name="text">
           <string>Right Dock Area</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DockCornerSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DockCornerSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
